test.sh

```diff
@@ -208,5 +208,5 @@ docker run --rm \
     -w /build \
     -t \
     -i \
-    mesosphere/dcos-commons:latest \
+    mesosphere/dcos-commons:uninstall_fix \
```

Comment:
Do we really need to do this?  If we're doing this in a real PR I'd say we can drop this, and on merge to master `mesosphere/dcos-commons:latest` will be appropriate.

Brings up a question.  Does our build use the published image or the one generated by the PR.  I think it should be the latter right?

Reply:
Reverted back to `master`.

No idea.